### DISCUSSION TITLE
pest++ question

### DISCUSSION THREAD

**Discussion by jbensabat** [usgs/pestpp#320] [Q&A]
Hello 
I am starting to use pest++ for the calibration of a surface flow model.
one of the options in the *.pst file is "forecasts".
in the doc it is stated that the arguments of forecasts can be provided also as a file.
there is no documentation about the structure of this file.
an example would be wecolme
thanks
jac

**Comment by jtwhite79** [comment]
just a simple one column ascii file with one observation per line should do it

**Comment by jbensabat** [comment]
thanks 
jac
